tests/invalid/Switch_Invalid_2.whiley

```diff
@@ -3,5 +3,4 @@ function f(int x) -> int:
         default:
             return 0
         case 1:
-            return -1
-    return 10
```

Comment:
Removed the return statement because the compiler now reports it as unreachable. The unreachable code error is correct, but this test case is meant to test the switch statement. Removing the return statement forces the compiler to return an error about the switch statement instead of about unreachable code.